lib/clickhousex/protocol.ex

```diff
@@ -165,15 +165,15 @@ defmodule Clickhousex.Protocol do
 
   ## Private functions
 
-  defp do_query(query, params, _opts, state) do
+  defp do_query(query, params, opts, state) do
```

Comment:
You can merge [e6b7cc5](https://github.com/Userpilot/clickhousex/commit/e6b7cc5a1ea7bb26d77af8fdae03952f662469af) from the main fork for this purpose